stories/Components/Atoms/FAAvatar/FAAvatar.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Stack } from "@mui/material";
+import { FAAvatar } from "@Atoms";
+
+import { IFAAvatarProps } from "@Atoms/FAAvatar/FAAvatar.interface";
+
+const story = {
+  title: "Atom/Avatar",
+  component: FAAvatar,
+  argTypes: {
+    testID: { control: "text" },
+    alt: { control: "text" },
+    size: { control: "text" },
+    color: { control: "text" },
+    src: { control: "text" },
+  },
+  args: {
+    testID: "avatar-storybook",
+    alt: "Avatar",
+    size: "medium",
+    color: "primary",
+    src: "https://mui.com/static/images/avatar/1.jpg",
+  },
+};
+
+export default story;
```

Comment:
Follow this https://github.com/HZN-one/flex/pull/63/files#r817334135 for consistency